test/checkpoint/workers.c

```diff
@@ -126,13 +126,22 @@ worker_op(WT_CURSOR *cursor, uint64_t keyno, u_int new_val)
 	char valuebuf[64];
 
 	cursor->set_key(cursor, keyno);
-	testutil_check(__wt_snprintf(
-	    valuebuf, sizeof(valuebuf), "%037u", new_val));
-	cursor->set_value(cursor, valuebuf);
-	if ((ret = cursor->insert(cursor)) != 0) {
-		if (ret == WT_ROLLBACK)
-			return (WT_ROLLBACK);
-		return (log_print_err("cursor.insert", ret, 1));
+	/* Roughly 5% removes. */
+	if (new_val % 19 == 0) {
+		if ((ret = cursor->remove(cursor)) != 0) {
+			if (ret == WT_ROLLBACK)
+				return (WT_ROLLBACK);
+			return (log_print_err("cursor.remove", ret, 1));
+		}
```

Comment:
The removes are also new functionality. AFAICT they're not related to online consistency checking. Should this change be part of this ticket/commit or is this part of some other work you were doing?

Reply:
This change was attempting to reproduce an issue from MongoDB that involves inconsistency between collection and indexes, which involved removes. I'll update the ticket and title to reflect that.